components/post_view/post_list_row/post_list_row.jsx

```diff
@@ -105,12 +119,16 @@ export default class PostListRow extends React.PureComponent {
             );
         }
 
-        if (listId === PostListRowListIds.OLDER_MESSAGES_LOADER || listId === PostListRowListIds.NEWER_MESSAGES_LOADER) {
+        const isOlderMessagesLoader = listId === OLDER_MESSAGES_LOADER;
+        const isNewerMessagesLoader = listId === NEWER_MESSAGES_LOADER;
+        if (isOlderMessagesLoader || isNewerMessagesLoader) {
+            const shouldHideAnimation = (isOlderMessagesLoader && !loadingOlderPosts) || (isNewerMessagesLoader && !loadingNewerPosts);
```

Comment:
you could simplify this as you already checked it should show the three dots
```suggestion
            const shouldHideAnimation = !loadingOlderPosts && !loadingNewerPosts;
```